src/commands/createNewProject/PythonProjectCreator.ts

```diff
@@ -287,23 +285,23 @@ function getVenvActivateCommand(venvName: string, platform: NodeJS.Platform): st
 }
 
 async function getPythonAlias(): Promise<string> {
-    for (const key of Object.keys(PythonAlias)) {
-        const alias: PythonAlias = <PythonAlias>PythonAlias[key];
-        const errorMessage: string | undefined = await validatePythonAlias(alias);
+    const aliasesToTry: string[] = ['python3.6', 'py -3.6', 'python3', 'python', 'py'];
+    const globalPythonPathSetting: string | undefined = getGlobalFuncExtensionSetting('pythonPath', 'python');
+    if (globalPythonPathSetting) {
+        aliasesToTry.unshift(globalPythonPathSetting);
```

Comment:
Is this why you got rid of the PythonAlias enum?

Reply:
Yeah I mean I also just find it easier to iterate through an array of strings rather than having to do the `Object.keys` thing.